src/seedu/addressbook/Main.java

```diff
@@ -105,17 +108,32 @@ private void recordResult(CommandResult result) {
      */
     private CommandResult executeCommand(Command command)  {
         try {
+        	storage.checkFileStillPresent();
```

Comment:
you need to check the presence/absence of the file in `save()` method, not in `Main`